include/os/linux/kernel/linux/dcache_compat.h

```diff
@@ -61,4 +61,25 @@ d_clear_d_op(struct dentry *dentry)
 	    DCACHE_OP_REVALIDATE | DCACHE_OP_DELETE);
 }
 
+/*
+ * Walk and invalidate all dentry aliases of an inode
+ * unless it's a mountpoint
+ */
+static inline void
+spl_d_drop_aliases(struct inode *inode)
```

Comment:
Let's rename this `zpl_d_drop_aliases()` per existing convention since this is part of the Linux posix layer abstraction and not the SPL.